flocker/testtools/test/test_amp.py

```diff
@@ -0,0 +1,85 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for :module:`flocker.testtools.amp`.
+"""
+
+from ..amp import FakeAMPClient, DelayedAMPClient
+
+from twisted.trial.unittest import SynchronousTestCase
+
+from twisted.protocols.amp import Command, Integer
+
+
+class TestCommand(Command):
+    """
+    Trivial command for testing.
+    """
+    arguments = [
+        ('argument', Integer()),
+    ]
+    response = [
+        ('response', Integer()),
+    ]
+
+
+class DelayedAMPClientTests(SynchronousTestCase):
```

Comment:
More unverified fakes.  Bummer.  Maybe someday we can stop writing such things and even come back and fix up these things we're accumulating.